layouts/default.vue

```diff
@@ -105,7 +103,7 @@ export default class Default extends Vue implements NuxtConfig {
   }
 
   head() {
-    const { htmlAttrs, meta } = this.$nuxtI18nSeo()
+    const { htmlAttrs, meta } = { addSeoAttributes: true } as MetaInfo
```

Comment:
```suggestion
    const { htmlAttrs, meta } = this.$nuxtI18nHead({ addSeoAttributes: true })
```
このように使うとよいらしいです（移行ガイド）